storm-core/src/clj/org/apache/storm/cluster_state/zookeeper_state_factory.clj

```diff
@@ -90,7 +91,7 @@
        (if (zk/exists zk-writer path false)
          (zk/set-data zk-writer path data)
          (do
-           (zk/mkdirs zk-writer (parent-path path) acls)
+           (zk/mkdirs zk-writer (Utils/parentPath path) acls)
```

Comment:
Like I said in #1045 there are some path manipulation functions that look like they belong in a Zookeeper.java file not in Utils.java.  Please work with them to figure out how to move them, even if it is in a follow on JIRA.